frontend/packages/dev-console/src/components/gitops/list/GitOpsList.tsx

```diff
@@ -1,36 +1,44 @@
 import * as React from 'react';
 import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
 import GitOpsListItem from './GitOpsListItem';
 import { Stack, StackItem, Split, SplitItem } from '@patternfly/react-core';
 import { GitOpsAppGroupData } from '../utils/gitops-types';
-import './GitOpsList.scss';
 import GitOpsEmptyState from '../GitOpsEmptyState';
+import './GitOpsList.scss';
 
 interface GitOpsListProps {
   appGroups: GitOpsAppGroupData[];
   emptyStateMsg: string;
 }
 
-const GitOpsList: React.FC<GitOpsListProps> = ({ appGroups, emptyStateMsg }) => (
-  <div className="odc-gitops-list">
-    {!emptyStateMsg ? (
-      <Stack hasGutter>
-        <StackItem>
-          <Split>
-            <SplitItem isFilled />
-            <SplitItem>{`${_.size(appGroups)} items`}</SplitItem>
-          </Split>
-        </StackItem>
-        {_.map(appGroups, (appGroup) => (
-          <StackItem key={`${appGroup.name}-${appGroup.repo_url}`}>
-            <GitOpsListItem appGroup={appGroup} />
+const GitOpsList: React.FC<GitOpsListProps> = ({ appGroups, emptyStateMsg }) => {
+  const { t } = useTranslation();
+  return (
+    <div className="odc-gitops-list">
+      {!emptyStateMsg ? (
+        <Stack hasGutter>
+          <StackItem>
+            <Split>
+              <SplitItem isFilled />
+              <SplitItem>
+                {t('devconsole~{{appGroupsSize, number}} item', {
+                  appGroupsSize: _.size(appGroups),
+                })}
```

Comment:
This is not generating the `_plural` in the `json` file.

I looked into the docs again, the pluralization only works if the variable is named `count`:
```suggestion
                {t('devconsole~{{count, number}} item', {
                  count: _.size(appGroups),
                })}
```
Sorry for the confusion.